.github/workflows/pre-build/expressjs.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Personally, I'd place this script under "expressjs" to have it close to the code that it's building. I'd place under .github/workflows only things common to all examples. But not a big deal.

Reply:
I think I'll leave it here because this file is CI-only. Putting it into a project folder feels off to me, as it isn't used anywhere in the project.

Reply:
OK